pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -91,6 +91,11 @@
                                     Condition="Exists('$(DotNetHostBinDir)/dotnet$(SymbolFileExtension)')">
       <IsSymbolFile>true</IsSymbolFile>
     </ArchitectureSpecificNativeFile>
+    <ArchitectureSpecificNativeFile Include="$(DotNetHostBinDir)/apphost$(ApplicationFileExtension)" />
```

Comment:
Few things:

- Apphost should never be part of SharedFX. If you look at Senthil's original PR, it was being explicitly deleted from being part of SharedFX. If it is part of SharedFX now, it is likely because of RID split package work that @ericstj was working upon.

- AppHost has nothing to do with SharedFX versioning and is only required to be part of Standalone publish. In this sense, it is sort of an artifact used only by SDK (or SDK experiences).

- Dotnet.exe versioning is the one that needs to be controlled with respect to the SharedFX we generate.